includes/discovery/sensors/runtime/apc.inc.php

```diff
@@ -15,3 +15,65 @@
     $high_limit         = 3000;
     discover_sensor($valid['sensor'], 'runtime', $device, $oid, $index, $type, $descr, $divisor, '1', $low_limit, $low_limit_warn, $warn_limit, $high_limit, $current);
 }
+
+// InRow IRRP100
+$oids = snmp_get($device, 'airIRRP100GroupSetpointsCoolMetric.0', '-OsqnU', 'PowerNet-MIB');
```

Comment:
Sorry I'm not sure what that means, the yaml will still record data like this does it's just abstracted away from writing code.